modules/ROOT/pages/policies-policy-categories.adoc

```diff
@@ -0,0 +1,40 @@
+= Policy Categories
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies are categorized by the function they perform. For example, because you are able to resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. The following table lists the Default policy name, its category and the function that it performs. +
+You can apply only one policy with a particular `providedCharacteristics` at a given time in an API.
+
+//Fede--what would 'providedCharacteristics' translate to for the customer?. I mean they need to understand what I'm talking about here right?
+
```

Comment:
I wonder if it would be more useful to the audience to have this table split into sections for each category of policy, rather than organized by policy names alphabetically.

Reply:
Great point! Organized by category. Done! Thanks!